### PR TITLE
Fixed a lot of sasslint warnings.

### PR DESCRIPTION
**Pull request by pmourelle**
🍰

### CODE REVIEW

File: src/modules/modals/_styles.scss [hunk 1]
```diff
@@ -1,16 +1,16 @@
 .mt3_modal-button {
+  @include respond-to-multiple(s, m) {
```
Comment (alexrqs) [inline src/modules/modals/_styles.scss]: Looks a little weird, the responsive mixins should be below the rules mostly when you target, s and m.

File: src/modules/videoplaylist/_styles.scss [hunk 2]
```diff
@@ -204,22 +205,14 @@ $animation-duration: 600ms;
...
   }
 
   &-container--thumbnails {
-    background-color: $mt3_color-black;
-    overflow: hidden;
-    transition-duration: 1s;
-    transition-property: max-height;
-    transition-timing-function: ease-in-out;
-
     @include respond-to(s) {
```
Comment (krlospelaez) [inline src/modules/videoplaylist/_styles.scss]: Shouldn't these `include`s be at the beginning, below `&--current-information` definition?

File: src/modules/videomodal/_styles.scss [hunk 3]
```diff
@@ -54,18 +53,18 @@ $mt3_color-inverse: $mt3_color-white;
   }
 
   &__title {
-    font-size: $mt3_h-med9 + (2 / 16);
-
     @include respond-to-multiple(s, m) {
       font-size: $mt3_h-sm6;
     }
+
+    font-size: $mt3_h-med9 + (2 / 16);
```
Comment (krlospelaez) [inline src/modules/videomodal/_styles.scss]: What 2 / 16 means? Maybe a var would be useful here?

File: src/modules/modals/_styles.scss (hunk 1 above)
Comment (pmourelle) [inline src/modules/modals/_styles.scss]: Agree. Sass-lint says mixins should be first. 
Maybe handling differently responsive stuff? Or ignoring sass-lint on such cases? Thoughts?

File: src/modules/videoplaylist/_styles.scss (hunk 2 above)
Comment (pmourelle) [inline src/modules/videoplaylist/_styles.scss]: Not following you here. There are inside of the same definition, just before any other property.

File: src/styles/themes/default/vars/_colors.scss [hunk 4]
```diff
@@ -17,21 +18,20 @@ $mt3_color-gray-2: #fafafa;
 $mt3_color-white: #fff;
 
 
+// Neutrals extended
+
+$mt3_color-white-semit: rgba($mt3_color-white, .5);
```
Comment (hkekeli) [inline src/styles/themes/default/vars/_colors.scss]: what is semit?

Comment (pmourelle) [inline src/styles/themes/default/vars/_colors.scss]: It'd stand for "semitransparent".

File: src/modules/modals/_styles.scss [hunk 5]
```diff
@@ -1,16 +1,16 @@
...
+    top: $mt3_unit - .0625;
+  }
+
   background-color: transparent;
   border: 0;
   color: $mt3_color-white;
   cursor: pointer;
   position: absolute;
   right: $mt3_unit-double - .125;
   top: $mt3_unit-double - .125;
```
Comment (arieldelfor) [inline src/modules/modals/_styles.scss]: These default rules are overriding the rules defined for the Small and Mobile breakpoints.
Move this rules into a new breakpoint or move ahead the previous breakpoint for M&S.

Also add the notes for QA. Let them know they should verify end ensure the bellow rules are being applied correctly:

```
.mt3_podpromo-content-container-top   .mt3_btn
   border: 0;
   color: $mt3_white;

.mt3_btn
    color: inherit;
    text-decoration: none;

.mt3_videopromo-button     .mt3_promocardtext--overlay-link 
    left: 0;
    z-index: zIndex('promocard-middle');
```

@pmourelle complete next values please:
```
.mt3_modal-button
   Breakpoint Mobile and Small
    top: 
    left: 
   Breakpoint Desktop
  top:
  lef:
```

Comment (pmourelle) [inline src/modules/modals/_styles.scss]: I reverted order for mixings implementing media queries, so they're now after properties being applied to their selectors.
I'll add a note for QA for checking on buttons when passing. Thanks.

**Comment by sebasrgl** [comment]
PR OK! @arieldelfor could you approve the changes?

**Pull request merged**